core/src/main/java/hudson/FilePath.java

```diff
@@ -2006,14 +1991,11 @@ public void copyTo(OutputStream os) throws IOException, InterruptedException {
         act(new SecureFileCallable<Void>() {
             private static final long serialVersionUID = 4088559042349254141L;
             public Void invoke(File f, VirtualChannel channel) throws IOException {
-                InputStream fis = null;
-                try {
-                    fis = Files.newInputStream(reading(f).toPath());
-                    Util.copyStream(fis,out);
+                try (InputStream fis = Files.newInputStream(reading(f).toPath())) {
+                    org.apache.commons.io.IOUtils.copy(fis, out);
                     return null;
                 } finally {
-                    org.apache.commons.io.IOUtils.closeQuietly(fis);
-                    org.apache.commons.io.IOUtils.closeQuietly(out);
+                    out.close();
```

Comment:
Maybe makes sense to indicate that the stream is automatically closed in Javadoc. It always confuses me. +1 for propagating exceptions if they happen